chat.delta.desktop.yml

```diff
@@ -21,6 +21,7 @@ finish-args:
   - --filesystem=xdg-videos
   - --share=network
   - --talk-name=org.freedesktop.Notifications
+  - --env=ELECTRON_OZONE_PLATFORM_HINT=auto
```

Comment:
And naturally this **doesn't work**, because it requires Electron 28.0.0 and upstream is on Electron 26.6.3

- https://www.electronjs.org/blog/electron-28-0
- https://github.com/deltachat/deltachat-desktop/blob/master/package-lock.json#L66

Reply:
and it's the default value, no? So I would be curious to learn why setting it is necessary.